server/utils/placementRequests/utils.ts

```diff
@@ -52,3 +54,30 @@ export const requestTypes = [
 
 export const withdrawalMessage = (duration: number, expectedArrivalDate: string) =>
   `Request for placement for ${placementLength(Number(duration))} starting on ${DateFormats.isoDateToUIDate(expectedArrivalDate, { format: 'short' })} withdrawn successfully`
+
+export const placementRequestTabItems = (activeTab?: string, apArea?: string, requestType?: string): Array<TabItem> => {
+  const hrefSuffix = createQueryString({ apArea, requestType })
```

Comment:
we should check if the return value from `createQueryString` is defined before appending it to the URL. That way we avoid adding `undefined`